developer-docs-site/docs/tutorials/run-a-fullnode.md

```diff
@@ -313,6 +313,25 @@ directly there).
 8. See the [Verify initial synchronization](#verify-initial-synchronization) section above for checking if the node
 is syncing again.
 
+## Node-Op FAQ
+
+This section will be updated over time. Feel free to add any Question/Answer pair.
+
+### Question(s):
+When starting the node, it throws a yaml-parsing error
+###  Answer:
+yaml files are very sensitive to formatting errors. Use a dedicated yaml editor and check if the each line is on the correct level
+
+---
+
+###  Question(s):
+I have no peers on the available node-testers  
```

Comment:
nit: add full stops for all questions. Also, change "node-testers" to "node testers".